From 822d3328bee88b9d2e2970b810af74192b722283 Mon Sep 17 00:00:00 2001
From: Jim Wilson <jimw@sifive.com>
Date: Mon, 22 Apr 2019 14:17:55 -0700
Subject: [PATCH 05/11] RISC-V: Enable 32-bit linux gdb core file support.

	bfd/
	* elfnn-riscv.c (PRSTATUS_SIZE) [ARCH_SIZE==32]: Change from 0 to 204.
---
 bfd/ChangeLog     | 4 ++++
 bfd/elfnn-riscv.c | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/bfd/ChangeLog b/bfd/ChangeLog
index 83e9e8ff37..0e00f14828 100644
--- a/bfd/ChangeLog
+++ b/bfd/ChangeLog
@@ -1,3 +1,7 @@
+2019-04-22  Jim Wilson  <jimw@sifive.com>
+
+	* elfnn-riscv.c (PRSTATUS_SIZE) [ARCH_SIZE==32]: Change from 0 to 204.
+
 2019-04-02  Jim Wilson  <jimw@sifive.com>
 
 	PR 24389
diff --git a/bfd/elfnn-riscv.c b/bfd/elfnn-riscv.c
index b7d82239fc..7f63a9b034 100644
--- a/bfd/elfnn-riscv.c
+++ b/bfd/elfnn-riscv.c
@@ -4008,7 +4008,7 @@ fail:
 }
 
 #if ARCH_SIZE == 32
-# define PRSTATUS_SIZE			0 /* FIXME */
+# define PRSTATUS_SIZE			204
 # define PRSTATUS_OFFSET_PR_CURSIG	12
 # define PRSTATUS_OFFSET_PR_PID		24
 # define PRSTATUS_OFFSET_PR_REG		72
-- 
2.20.1

